dp/protos/BUILD.bazel

```diff
@@ -28,3 +28,18 @@ python_grpc_library(
     name = "enodebd_dp_python_grpc",
     protos = [":enodebd_dp_proto"],
 )
+
+proto_library(
+    name = "cbsd_proto",
+    srcs = ["cbsd.proto"],
+)
+
+python_proto_library(
+    name = "cbsd_python_proto",
+    protos = [":cbsd_proto"],
+)
+
+python_grpc_library(
```

Comment:
The `cbsd_python_grpc` target is not used anywhere, can we remove it from the PR?

Reply:
@LKreutzer 
It is a mistake. I am trying to fix the BAZEL builds, and I know very little about BAZEL so with each new commit I am looking at BAZEL builds to see if I made an improvement.

The `cbsd_python_grpc` is now being used.